docs/en/ingest-management/elastic-agent/elastic-agent-providers.asciidoc

```diff
@@ -201,12 +197,10 @@ In this, this option should be carefully considered.
 Provides the option to enable leaderelection between a set of Agents
```

Comment:
Is leaderelection one word or two?

Reply:
leaderelection looks to be one word to me.

In general that's correct. Elastic Agent is the product name in this instance, so we want to capitalize all instances of Agent.

Reply:
We actually have an attribute for this. All instances of "Agent" can be replaced with `{agent}`. For the plural form, you can use `{agent}'s`. This will substitute our capitalized product name automatically.